dna/examine.go

```diff
@@ -20,15 +23,15 @@ func Count(seq []Base) (ACount int, CCount int, GCount int, TCount int, NCount i
 			TCount++
 		case N:
 			NCount++
-		case a:
+		case LowerA:
```

Comment:
This name change is nice so that lowercase A can be seen outside of the dna package.